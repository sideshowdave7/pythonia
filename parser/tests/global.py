globalvar = 0

def foo():
  global globalvar
  globalvar = 10

foo()

print(globalvar)
