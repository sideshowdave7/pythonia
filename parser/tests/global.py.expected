(program
 (= (globalvar) 0)
 (def (foo) (suite (global globalvar) (= (globalvar) 10)))
 (expr (indexed foo (called)))
 (expr (indexed print (called globalvar))))

